content/200-concepts/100-components/02-prisma-client/037-relation-queries.mdx

```diff
@@ -613,12 +534,12 @@ The following query uses a nested [`createMany`](/reference/api-reference/prisma
 - One user
 - Two posts
 
-The example uses a nested `include` to include all posts. Note that it is **not possible** to nest an additional `create` or `createMany` inside the highlighted query, which means that you cannot create a user, posts, and post categories at the same time:
+The example uses a nested `include` to include all posts.
 
 <CodeWithResult>
 <cmd>
 
-```ts highlight=5;normal
+```ts highlight=4-11;normal
```

Comment:
this seems glitchy:

<img width="527" alt="image" src="https://github.com/prisma/docs/assets/4058327/69fe2e8f-ee56-4f85-b75e-a7870c570582">

Reply:
I think I know what's going on here. The recommit hook is formatting the codeblock as well.

Initially, the objects in the array were on multiple lines for ease of readability on slightly smaller screens but I'll update the code formatting to play nicely with our recommit hook